src/components/OmouComponents/BackgroundPaper.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import Paper from "@material-ui/core/Paper";
+
+export default function BackgroundPaper({children, ...rest}) {
+	return (<Paper style={{"padding": "3%"}} {...rest}>
```

Comment:
Does this need `className="paper"` or not?

Reply:
eh..... I want to delete that class. I think it's too confusing between the `<Paper>` component and this `<BackgroundPaper>` component.